interview-questions.md

```diff
@@ -8,58 +8,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. The built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: I'm honestly not sure. I'm guessing arrays.
 
-  Researched answer:
+  Researched answer: .map() parameters are value, indez and array.
 
 
 
 2. What is object destructuring?
 
-  Your answer:
+  Your answer: To break down in segments?
 
-  Researched answer:
+  Researched answer:It is taking variables and assigning them in a specific way.
 
 
 
 3. Why would you use the method super()?
 
-  Your answer:
+  Your answer: I dont remember but it has something to do with classes.
 
-  Researched answer:
+  Researched answer: super() passes information from the contructor.
 
 
 
 4. What is a React Fragment? Why would you use one?
 
-  Your answer:
+  Your answer: I don't remember
 
-  Researched answer:
+  Researched answer: It's what you use to next other components.
 
 
 
 5. What is JSX?
 
-  Your answer:
+  Your answer: Something that javascript uses to allow it to work within a browser?
 
-  Researched answer:
+  Researched answer: It's Javascripts version of HTML
 
 
 
 6. STRETCH: What is hoisting in JavaScript?
 
-  Your answer:
+  Your answer: moving code to a different location.
 
-  Researched answer:
+  Researched answer: Moving variables and functions to the top of the code.
```

Comment:
I appreciate that you answered these interview questions honestly. Not knowing an answer is okay in an interview but let's practice setting yourself apart by your ability to show you can find the material that you know.  

What is a React Fragment? Why would you use one?  Your answer:  

-While I know it has something to do with JSX I can't recall what it is specifically. I would probably look at the react documentation and determine what it is and how to use it. -

or

-I know what react fragment's look like <> </> but I don't know what they are- 

something along those lines...